templates/global/quantity-input.php

```diff
@@ -43,6 +43,8 @@ class="<?php echo esc_attr( join( ' ', (array) $classes ) ); ?>"
 			inputmode="<?php echo esc_attr( $inputmode ); ?>"
 			<?php if ( ! empty( $labelledby ) ) { ?>
 			aria-labelledby="<?php echo esc_attr( $labelledby ); ?>" />
+			<?php } else { ?>
```

Comment:
Instead of adding an else, you'd just need to move the above `/>` out of the if statement.

Reply:
@mikejolley I've updated the condition as per your suggestion :)